### PR TITLE
Remove unused `npm` dependencies from website

### PR DESCRIPTION
**Pull request by lgeiger**
This PR removes a few unused `npm` dependencies from the website requirements.

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -62,7 +60,6 @@
     "next-auth": "^4.20.1",
     "next-i18next": "^13.2.2",
     "nodemailer": "^6.9.1",
-    "npm": "^9.6.2",
```
Comment (lgeiger) [inline website/package.json:65]: `npm` is meant to be installed globally, so there is no need to include it in `package.json`

File: website/package.json [hunk 2]
```diff
@@ -42,13 +42,11 @@
     "@prisma/client": "^4.11.0",
     "@tailwindcss/forms": "^0.5.3",
     "@tanstack/react-table": "^8.8.2",
-    "accept-language-parser": "^1.5.0",
     "autoprefixer": "^10.4.14",
     "axios": "^1.3.4",
     "boolean": "^3.2.0",
     "chart.js": "^4.2.1",
     "clsx": "^1.2.1",
-    "cookies": "^0.8.0",
```
Comment (lgeiger) [inline website/package.json:51]: This dependency was added in #1893 but appears to be unused now.

File: website/package.json [hunk 3]
```diff
@@ -42,13 +42,11 @@
     "@prisma/client": "^4.11.0",
     "@tailwindcss/forms": "^0.5.3",
     "@tanstack/react-table": "^8.8.2",
-    "accept-language-parser": "^1.5.0",
```
Comment (lgeiger) [inline website/package.json:45]: This dependency was added in #881 but appears to be unused now.

**Review by notmd — APPROVED** [review]
Thanks, can you resolve the conflict?

**Pull request merged**